kong/plugins/prometheus/exporter.lua

```diff
@@ -58,6 +63,15 @@ local function init()
                                          {"type", "service", "route"})
 end
 
+local function init_worker()
+  local err
+  -- create a lua-resty-counter instance with sync interval of every second
+  counter_instance, err = counter.new("prometheus_metrics", 1)
+  if err then
+    error(err)
+  end
+  prometheus:set_resty_counter(counter_instance)
+end
```

Comment:
I'm sure there is a reason for the `init_worker()` code to not be a part of the `init()` above. What is it?

Reply:
`init()` is called during master config parsing, but we will need the lua-resty-counter instance to be `new()`ed in each worker so they can have a timer.